README.md

```diff
@@ -1,2 +1,12 @@
 # MyL3-External-Data
 GSoC 2018 project
+
+Idea is to develop basic Infusion wrapper for My Life Long Learning Lab(MyL3). This wrapper will contain following data about learners imported from various external sources.
```

Comment:
I would maybe say something like:

"This project aims to develop external data sources for the My Life Long Learning Lab (MyL3) project.

Data sources that will be researched and prototyped include:"

I don't think I would describe the project as a "wrapper".

Reply:
Updated